cc_licenses/templates/includes/deed_cc0_body.html

```diff
@@ -0,0 +1,62 @@
+          <li><p class=" body-big padding-bottom-normal">{% blocktrans trimmed %}When using or citing the work, you should not imply <a href="" class="helpLink" id="endorsement">endorsement</a> by the author or the affirmer.{% endblocktrans %}</li>
+        </ul>
+      </div>
+    </div>
+
+    <h4 class="b-header  padding-bottom-big padding-top-normal" style="font-weight: bold;">{% blocktrans trimmed %}Other Rights{% endblocktrans %}</h4>
+    <div>
+      <p>{% blocktrans trimmed %}The use of a work free of known copyright restrictions may be otherwise regulated or limited. The work or its use may be subject to personal data protection laws, publicity, image, or privacy rights that allow a person to control how their voice, image or likeness is used, or other restrictions or limitations under applicable law.{% endblocktrans %}</p>
+      <p><a href="https://wiki.creativecommons.org/Frequently_Asked_Questions#When_are_publicity_rights_relevant.3F">{% blocktrans trimmed %}Learn more{% endblocktrans %}</a>.</p>
+    </div>
+
+
+    <h4 class="b-header  padding-bottom-big padding-top-normal" style="font-weight: bold;">{% blocktrans trimmed %}Endorsement{% endblocktrans %}</h4>
+    <div>
+      <p>{% blocktrans trimmed %}In some jurisdictions, wrongfully implying that an author, publisher or anyone else endorses your use of a work may be unlawful.{% endblocktrans %}</p>
+      <p><a href="https://wiki.creativecommons.org/Frequently_Asked_Questions#Do_I_need_to_be_aware_of_anything_else_when_providing_attribution_or_credit.3F">{% blocktrans trimmed %}Learn more{% endblocktrans %}</a>.</p>
+    </div>
+
+    <h4 class="b-header  padding-bottom-big padding-top-normal" style="font-weight: bold;">{% blocktrans trimmed %}Citation{% endblocktrans %}</h4>
+    <div>
+      <p>
+        {% blocktrans trimmed %}Copy and paste the HTML provided into your webpage to easily cite this work.{% endblocktrans %}
+      </p>
+      <p>
+        <a href="https://wiki.creativecommons.org/CC0_FAQ#Do_I_have_to_attribute_the_person_who_applied_CC0_to_their_work.3F">{% blocktrans trimmed %}Learn more{% endblocktrans %}</a>
+      </p>
+    </div>
+
+    <h4 class="b-header  padding-bottom-big padding-top-normal" style="font-weight: bold;">{% blocktrans trimmed %}Who is the affirmer?{% endblocktrans %}</h4>
+    <div>
+      <p>
+        {% blocktrans trimmed %}The affirmer is the person who surrendered rights to the work worldwide using CC0, to the extent allowable by law. It may be the original author of the work or another person who may have had some copyright or related or neighboring legal rights in the work.{% endblocktrans %}
+      </p>
+    </div>
```

Comment:
The spacing/alignment of these sections is off. Shouldn't these sections be laid out similarly to `Other Rights`?

Reply:
Got the div nesting wrong, should be fixed when I push my latest changes.